### ISSUE TITLE
Function List - Which Flavor?

### ISSUE THREAD

**Issue by shawnoden** [foundeo/cfdocs#421]
On reports/todo.cfm >> in funcList, are the functions listed Adobe only, or should the Lucee functions be in there too? I'm showing 722 total functions currently documented, but only 609 in CF2016. It looks like the Missing Functions are pulled from funcList, which doesn't seem to include non-Adobe functions. Is that true? It does look like all the Lucee-specific functions are listed (but I guess I need to check that).  My local, with all functions included, shows that all but 29 functions are documented (with all Lucee done).

**Comment by pfreitag** [comment]
Hey Shawn - yes I think that missing functions report is only looking at ACF functions from a list generated using `getFunctionList()` on CF11... A bunch of Lucee specific functions were added at some point, so that would show up in the overall total. There also may be some that are documented but have since been removed so they would not show up in `getFunctionList()`

**Comment by shawnoden** [comment]
Good deal. It looks like there are a bunch of other functions added since that list was last updated. See my totals above. If you want, I can update that list to include the current ACF2016 and Lucee functions. The Lucee functions already seem to be in the docs.

**Comment by shawnoden** [comment]
And I'll try no to screw up my pull request again. Not sure why it went to Foundeo. I thought I had it going to my local repo. :-S

**Comment by pfreitag** [comment]
Yes if you wouldn't mind updating it for CF2016 that would be great.

**Comment by shawnoden** [comment]
Cool deal. I'll take care of that this evening.
